app/assess/templates/assesser_project_view.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/details/macro.html' import govukDetails -%}
+{%- from 'govuk_frontend_jinja/components/back-link/macro.html' import govukBackLink -%}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+
+{% block beforeContent %}
+{{
+  govukBackLink({'href': url_for("assess_bp.fund_round", fund_id=fund_data.identifier, round_id=round_data.identifier)})
+}}
+{% endblock %}
+
+
+{% block content %}
+
+<span class="govuk-caption-l">Project reference</span>
+<h1 class="govuk-heading-l">{{application_data.identifier}}</h1>
+
+    {{ govukDetails({
+        'summaryText': fund_data.name,
+        'text': fund_data.description,
+        'id': "assessment-page-details",
+        'open':True}) }}
+
+        <script nonce="{{ csp_nonce() }}">
```

Comment:
Nice nonce! One thought though - as this is an accessibility feature, we'll probably need it on all govukDetails components right? So perhaps could this script be more global and go in the footer, perhaps in a document.onLoad or similar and auto-close all elements with the govuk-details classname maybe?

Reply:
Great spot Harry! I will move it into the head.